packages/docs/getting-started/running-a-validator.md

```diff
@@ -91,26 +105,27 @@ Note that account and signer keys must be unique and may not be reused.
-| DATABASE_URL                         | The URL under which your database is accessible, currently supported are `postgres://`, `mysql://` and `sqlite://` |
-| APP_SIGNATURE                        | The hash with which clients can auto-read SMS messages on android                                                  |  |
-| SMS_PROVIDERS                        | A comma-separated list of providers you want to configure, we currently support `nexmo` & `twilio`                 |
+| Variable                             | Explanation                                                                                                                          |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
+| CELO_IMAGE                           | The Docker image used for the Validator and Proxy containers                                                                         |  |
+| NETWORK_ID                           | The Celo Baklava network chain ID                                                                                                    |  |
+| CELO_VALIDATOR_GROUP_ADDRESS         | The account address for the Validator Group                                                                                          |  |
+| CELO_VALIDATOR_ADDRESS               | The account address for the Validator                                                                                                |  |
+| CELO_VALIDATOR_SIGNER_ADDRESS        | The address of the validator signer authorized by the validator account                                                              |  |
+| CELO_VALIDATOR_SIGNER_PUBLIC_KEY     | The ECDSA public key associated with the validator signer address                                                                    |  |
+| CELO_VALIDATOR_SIGNER_SIGNATURE      | The proof-of-possession of the validator signer key                                                                                  |  |
+| CELO_VALIDATOR_SIGNER_BLS_PUBLIC_KEY | The BLS public key for the Validator instance                                                                                        |  |
+| CELO_VALIDATOR_SIGNER_BLS_SIGNATURE  | A proof-of-possession of the BLS public key                                                                                          |  |
+| PROXY_ENODE                          | The enode address for the Validator proxy                                                                                            |  |
+| PROXY_INTERNAL_IP                    | (Optional) The internal IP address over which your Validator can communicate with your Proxy                                         |  |
+| PROXY_EXTERNAL_IP                    | The external IP address of the Proxy. May be used by the Validator to communicate with the Proxy if PROXY_INTERNAL_IP is unspecified |  |
```

Comment:
Maybe we can add that people can test their external IP address by running the following command:

`dig +short myip.opendns.com @resolver1.opendns.com`

Reply:
There are instructions below to use telnet, is `dig` preferable?

Reply:
Telnet is good at checking whether your port is open, but this command tells you what your external IP is.

Reply:
Ah gotcha, perfect, thanks.